CMakeLists.txt

```diff
@@ -19,11 +19,6 @@ if (IS_BIG_ENDIAN)
 	message(FATAL_ERROR "${PROJECT_NAME} currently does not support big endian systems.")
 endif()
 
-option(LLL "Build LLL" OFF)
-option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
```

Comment:
Why is this removed?

Reply:
To validate that you guys are still reviewing properly.

Reply:
My morning reviews are proper

Reply:
It is already the afternoon.